PureBasicIDE/ProjectManagement.pb

```diff
@@ -2166,10 +2178,18 @@ Procedure ProjectOptionsEvents(EventID)
             ; switch to the info source, so the user can directly get to the projects compiler options etc
             FirstElement(FileList())
             ChangeActiveSourceCode()
+          ElseIf OldProjectName$ <> ProjectName$
+            ;change name of Tab, if user changed the project name
+            ;Tab can be moved by customer, therefore going through all open Tabs to find it. Maybe there is a better solution, but it works
+            For i = 0 To CountTabBarGadgetItems(#GADGET_FilesPanel) - 1
+              If GetTabBarGadgetItemText(#GADGET_FilesPanel, i) = Language("Project", "TabTitle") + " [" + OldProjectName$ + "]"
+                SetTabBarGadgetItemText(#GADGET_FilesPanel, i, Language("Project", "TabTitle") + " [" + ProjectName$ + "]")
```

Comment:
You need to apply the change with the [project name] also in line 568 of `ProjectManagement.pb` (that code is called when preferences like language have changed)

That line also shows how to figure out the Tab index of the project tab without a loop and comparing element names. I think this would be a more elegant solution for this update here as well.